src/pal/src/init/pal.cpp

```diff
@@ -537,8 +537,6 @@ Initialize(
     SEHCleanup();
 CLEANUP5:
     PROCCleanupInitialProcess();
-CLEANUP4:
```

Comment:
Do these numbers mean something? If we're going to keep the various CLEANUP labels here, should we renumber them so they are consecutive?